Ambit/Source/Ambit/Mode/ConfigImportExport.h

```diff
@@ -137,6 +146,16 @@ class UConfigImportExport : public UObject
     TFunction<FString(const FString& FileExtension, const FString& DefaultPath, const FString& Filename)>
     LambdaGetPathFromPopup = AmbitFileHelpers::GetPathForFileFromPopup;
 
+    /**
+     * Calls UGltfExport::Export()
+     * Allows for injection of the function to be changed. Should only be changed in testing.
+     */
+    TFunction<UGltfExport::GltfExportReturnCode(UWorld* WorldContext, const FString& FilePath)> ExportGltf = [this](
+        UWorld* WorldContext, const FString& FilePath)
+    {
+        return this->GetGltfExporter()->Export(WorldContext, FilePath);
```

Comment:
This looks like it needs error handling.

Reply:
done. resolving conversation since PR is approved